recipes/osmesa/meta.yaml

```diff
@@ -0,0 +1,31 @@
+{% set version = "11.2.2" %}
+
+package:
+  name: osmesa
+  version: {{ version }}
+
+source:
+  url: https://github.com/mesa3d/mesa/archive/mesa-{{ version }}.tar.gz
+  fn: mesa-v{{ version }}.tar.gz
+  sha256: a2edf7ad2589f0c3b5cd3d8f8866193c03bf871a38d5dc27423a03f352abe2de
+
+build:
+  number: 0
+  # feel free to figure it out and submit a PR
+  # instructions here: http://mesa3d.org/install.html
```

Comment:
The comment is fine, but ideally I would like to see an issue on the feedstock once it is created.